app/javascript/packages/document-capture/components/in-person-full-address-search.tsx

```diff
@@ -226,6 +241,8 @@ function FullAddressSearch({
           onChange={onZipCodeChange}
           label={t('in_person_proofing.body.location.po_search.zipcode_label')}
           disabled={disabled}
+          pattern="(\d{5}([\-]\d{4})?)"
+          maxLength={10}
         />
```

Comment:
5 numbers + dash + 4 numbers = maxLength of 10. The [IPPaaS Web Services ICD docs](https://docs.google.com/document/d/1NEbiA5QlOZMxFFCuUswp-3qsiaII7Zwc/edit) say it will take 11 digits but all of the validation I have found uses this pattern xxxxx-xxxxx.